src/providers/PersistRehydrationProvider.tsx

```
@@ -69,6 +69,16 @@ function QueryPersistSyncGate({
   return <>{children}</>;
 }
 
+/** persist 대상이 없을 때 — 마운트 직후 1회 콜백 (Zustand 게이트와 동등한 “준비됨” 시점) */
+function RunOnceOnMount({ fn }: { fn?: () => void }) {
+  const fnRef = useRef(fn);
+  fnRef.current = fn;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/providers/PersistRehydrationProvider.tsx:75:3
  73 | function RunOnceOnMount({ fn }: { fn?: () => void }) {
  74 |   const fnRef = useRef(fn);
> 75 |   fnRef.current = fn;
     |   ^^^^^^^^^^^^^ Cannot update ref during render
  76 |   useEffect(() => {
  77 |     fnRef.current?.();
  78 |   }, []);

(react-hooks/refs)